third_party/nvfuser/csrc/executor_kernel_arg.cpp

```diff
@@ -152,6 +152,13 @@ KernelArgumentHolder KernelArgumentHolder::createKernelArgumentHolder(
 
 // Push a tensor to the arguments
 void KernelArgumentHolder::push(const at::Tensor& tensor) {
+  // Deferred update of index_mode.
+  // Index mode size depends on the fusion intputs AND outputs.
```

Comment:
This should actually depend on all global memory buffers. Inputs and outputs need to take into consideration of stride, but even intermediate buffers allocated on global memory could impact this decision.

Reply:
So it's just the comment that is wrong? Or I should check for sizes in other places too?

Reply:
I think inputs and outputs are okay to check now, but please add a todo.

Reply:
@mmigdal-nv Can you please create a github issue to track this?